lib/rabbit_feed/consumer_connection.rb

```diff
@@ -79,6 +79,7 @@ def handle_message delivery_info, payload, &block
         acknowledge delivery_info
       rescue => e
         handle_processing_exception delivery_info, e
+        raise if RabbitFeed.configuration.consumer_exit_after_fail
```

Comment:
was thinking to exit(1) but then it will not to do `ensure` code in 
https://github.com/simplybusiness/rabbit_feed/blob/master/lib/rabbit_feed/consumer_connection.rb#L42-L44